src/backends/torch/torchdataset.h

```diff
@@ -79,6 +79,10 @@ namespace dd
         = nullptr;               /**< back ptr to input connector. */
     bool _classification = true; /**< whether a classification dataset. */
 
+    std::default_random_engine _rnd_generator;
```

Comment:
where is this used?

Reply:
Good catch, it's a leak from my data augmentation branch, will remove.

Reply:
Done.